include/SFML/Graphics/Color.inl

```diff
@@ -73,7 +73,7 @@ constexpr Color operator+(const Color& left, const Color& right)
 {
     const auto clampedAdd = [](std::uint8_t lhs, std::uint8_t rhs) -> std::uint8_t
     {
-        const int intResult = static_cast<int>(lhs) + static_cast<int>(rhs);
+        const int intResult = int{lhs} + int{rhs};
```

Comment:
I'd vote to keep the static_cast. It's clearer what's going on and I don't see the gain in saving a few characters.

Reply:
The problem is, a static cast is more than we need. A static cast signals that we may be doing a narrowing conversion which in this case we don't need so not only are we making the code shorter, but we're also correctly expressing how this conversion ought not discard any information.

I looked through the C++ Core Guidelines for guidance and it seems to also recommend using `{}` in these cases where no narrowing is occurring. I believe using `{}` in this case is technically not even a cast but rather treated as construction.

https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Res-casts
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Res-casts-named
<img width="833" alt="Screenshot 2022-12-30 at 11 06 23 AM" src="https://user-images.githubusercontent.com/39244355/210100003-2e117401-d632-42e5-afe6-bef40151a5a9.png">

---

While I acknowledge that arguing over this particular example is not productive, this is a precedent I think is worth setting for the whole library since I believe we all agree that we ought to do as little casting as possible.